addons/mail/controllers/link_preview.py

```diff
@@ -23,9 +23,9 @@ def mail_link_preview(self, message_id):
 
     @http.route("/mail/link_preview/hide", methods=["POST"], type="jsonrpc", auth="public")
     @add_guest_to_context
-    def mail_link_preview_hide(self, link_preview_ids):
+    def mail_link_preview_hide(self, link_preview_message_ids):
         guest = request.env["mail.guest"]._get_guest_from_context()
-        link_preview_sudo = guest.env["mail.link.preview"].sudo().search([("id", "in", link_preview_ids)])
+        link_preview_sudo = guest.env["mail.link.preview.message"].sudo().browse(link_preview_message_ids)
```

Comment:
sudo comment would be helpful as actual access check is done 4 lines below.

Reply:
Agree, I forgot to justify this sudo